content-src/components/Search/Search.js

```diff
@@ -7,13 +7,13 @@ const Search = React.createClass({
     };
   },
   render() {
-    return (<div className="search-wrapper">
+    return (<form className="search-wrapper">
       <span className="search-label fa fa-search" />
-      <input ref="input" placeholder="Search" required />
+      <input ref="input" type="search" placeholder="Search" required />
```

Comment:
I'm a bit confused here. Even though we specify `required`, I can leave the field blank and click Enter or press the arrow icon and I get a brief tooltip pop up saying "Please fill out this field." and then the form immediately submits to Yahoo!.